token/js/module.flow.js

```diff
@@ -102,6 +112,7 @@ declare module '@solana/spl-token' {
     createMultisig(m: number, signers: Array<PublicKey>): Promise<PublicKey>;
     getMintInfo(): Promise<MintInfo>;
     getAccountInfo(account: PublicKey): Promise<AccountInfo>;
+    getOrCreateAssociatedAccountInfo(account: PublicKey): Promise<AccountInfo>;
```

Comment:
```suggestion
    getOrCreateAssociatedAccountInfo(owner: PublicKey): Promise<AccountInfo>;
```
small thing, but confused me a little

Reply:
done!